frontend/package.json

```diff
@@ -3,8 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "axios": "^0.19.0",
```

Comment:
You'll probably need `axios` later, though

Reply:
Ok, reinstalling axios now

Reply:
Fixed!